components/SubmissionsTable/SubmissionRow.tsx

```diff
@@ -0,0 +1,135 @@
+import { Submission } from 'data/flexibleForms/forms.types';
+import Link from 'next/link';
+import { format } from 'date-fns';
+import forms from 'data/flexibleForms';
+import { User } from 'types';
+import s from './index.module.scss';
+
+interface Props {
+  submission: Submission;
+  openRow: string | false;
+  setOpenRow: (value: string | false) => void;
+  user: User;
+}
+
+const SubmissionRow = ({
+  submission,
+  openRow,
+  setOpenRow,
+  user,
+}: Props): React.ReactElement => {
+  const open = openRow === submission.submissionId;
+
+  const form = forms.find((form) => form.id === submission.formId);
+
+  const completedSteps = Object.keys(submission.formAnswers).length;
+  const totalSteps = form?.steps?.length;
+
+  const lastEdited =
+    submission.editHistory[submission.editHistory.length - 1]?.editTime;
+
+  const editors = submission.workers
+    .map((worker) => worker.email)
+    .filter((editor) => editor !== user.email);
+
+  return (
+    <>
+      <li className={s.row}>
+        <div className={s.person}>
+          <h3 className={s.name}>
+            <Link href={`/people/${submission.residents[0].id}`}>
```

Comment:
Below you check if `residents` is null before accessing the first item, but you don't here? Is that a type issue, a mistake, or something else?

Reply:
the proper shape of this will be determined once we do group recording, but safe to assume it'll always be defined for now